fs/fs.go

```diff
@@ -194,7 +194,7 @@ type filesystem struct {
 	metricsController     *layermetrics.Controller
 	attrTimeout           time.Duration
 	entryTimeout          time.Duration
-	sociIndex             *soci.SociIndex
+	sociIndex             *soci.Index
```

Comment:
what is this change for?

Reply:
Just a rename. `soci.SociIndex` seemed redundant. And the linter (and me) gets slightly annoyed. Hence this comment: https://github.com/awslabs/soci-snapshotter/blob/main/soci/soci_index.go#L60